discord/interaction.go

```diff
@@ -365,6 +368,19 @@ func (o CommandInteractionOption) FloatValue() (float64, error) {
 	return f, err
 }
 
+// ModalInteraction is the submitted modal form
+type ModalInteraction struct {
+	CustomID   ComponentID          `json:"custom_id"`
+	Components *ContainerComponents `json:"components,omitempty"`
```

Comment:
Is this `null`able? Do we need the pointer type?

Reply:
Yeah my mistake, Modals require an array of components so let me fix that.